source/Player.cpp

```diff
@@ -59,9 +59,9 @@ namespace GTA
 	{
 		return Native::Function::Call<bool>(Native::Hash::IS_PLAYER_FREE_AIMING, this->Handle);
 	}
-	bool Player::IsOnMission::get()
+	bool Player::CanStartMission::get()
```

Comment:
Just saw the exclamation mark, wasn't inverted, my bad.

So any reason to actually change the name? Is there any case where `CAN_PLAYER_START_MISSION` returns false with not being in a mission?